RoutesApp_ios/modules/home/controllers/HomeViewController.swift

```diff
@@ -192,6 +201,20 @@ class HomeViewController: UIViewController {
         }
     }
 
+    private func gotToTab() {
+        destinationContainer.isHidden = true
+        let tabViewController = SceneDelegate.shared?.window?.rootViewController as? UITabBarController
+        guard let tabController = tabViewController else { return }
+        switch destinationFromDifferentController {
+        case .TOURPOINTS:
+            tabController.selectedIndex = 1
```

Comment:
use const values to indicate which tab is which on the selectedIndex

Reply:
Changed for const values